src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -119,53 +119,57 @@ var editCommentHandler = function(e) {
+            beforeSend: function() {
+                submitButton.html("<img src='/images/ajax-loader.gif'/>");
+            },
+            error: function() {
                 if (editForm.is(':visible')) {
-                    setFormErrorMessage(editForm.find('div > a'), data.errorMessage);
+                    setFormErrorMessage(editForm.find('div > a'), 'Failed to delete comment. Please try again.');
                 } else if (frCommentList.parent().find('div.delete_error_msg').length === 0) {
-                    frCommentList.after('<div class="delete_error_msg alert alert-danger">' + data.errorMessage + '</div>');
+                    frCommentList.after('<div class="delete_error_msg alert alert-danger">'
+                                        + 'Failed to delete comment. Please try again.</div>');
                 }
                 submitButton.html('<span class="glyphicon glyphicon-trash glyphicon-primary"></span>');
-            } else if (isInCommentsPage()) {
-                reloadFeedbackResponseComments(formObject, panelHeading);
-            } else {
-                var numberOfItemInFrCommentList = deletedCommentRow.parent().children('li');
-                if (numberOfItemInFrCommentList.length <= 2) {
-                    deletedCommentRow.parent().hide();
-                }
-                if (frCommentList.find('li').length <= 1) {
-                    frCommentList.hide();
+            },
+            success: function(data) {
+                if (data.isError) {
+                    if (editForm.is(':visible')) {
+                        setFormErrorMessage(editForm.find('div > a'), data.errorMessage);
+                    } else if (frCommentList.parent().find('div.delete_error_msg').length === 0) {
+                        frCommentList.after('<div class="delete_error_msg alert alert-danger">'
+                                + data.errorMessage + '</div>');
+                    }
+                    submitButton.html('<span class="glyphicon glyphicon-trash glyphicon-primary"></span>');
+                } else if (isInCommentsPage()) {
+                    reloadFeedbackResponseComments(formObject, panelHeading);
+                } else {
+                    var numberOfItemInFrCommentList = deletedCommentRow.parent().children('li');
+                    if (numberOfItemInFrCommentList.length <= 2) {
+                        deletedCommentRow.parent().hide();
+                    }
+                    if (frCommentList.find('li').length <= 1) {
+                        frCommentList.hide();
+                    }
+                    deletedCommentRow.remove();
+                    frCommentList.parent().find('div.delete_error_msg').remove();
                 }
-                deletedCommentRow.remove();
-                frCommentList.parent().find('div.delete_error_msg').remove();
             }
-        }
+        });
     });
+    return false;
```

Comment:
Actually that's because the `preventDefault()` above was at the wrong level, now this `return false` should not be necessary anymore.